src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -19,4 +19,31 @@ void fooTest() {
         assertEquals(40, app.items[0].quality);
     }
 
+
+   @Test
+   void agedBrieTest() {
+       Item[] items = new Item[] { new Item("Aged Brie", 5, 0) };
+       GildedRose gildedRose = new GildedRose(items);
+       gildedRose.updateQuality();
+
+       assertEquals("Aged Brie", gildedRose.items[0].name);
+       assertEquals(4, gildedRose.items[0].sellIn);
+       assertEquals(1, gildedRose.items[0].quality);
+   }
+
+    @Test
+    void negativeQuality() {
```

Comment:
naming: I felt from the test name that we expect a negative quality.  adding something like @DispalyName(Quality cannot be negative) or renaming the test to quality_should_not_be_negative can be a better description

Reply:
Done